html/pfappserver/lib/pfappserver/PacketFence/Controller/Graph.pm

```diff
@@ -907,6 +907,141 @@ sub osclassbandwidth :Local :AdminRole('REPORTS') {
     $self->_add_links($c, 'dhcp_fingerprint', 'equal', 'label');
 }
 
+=head2 topauthenticationfailures_by_mac
+
+Radius AAA errors by mac for a specific period.
+
+Defined as a report.
+
+=cut
+
+sub topauthenticationfailures_by_mac :Local :AdminRole('REPORTS') {
```

Comment:
This naming convention makes my eyes bleed. IDK if I'm the only one

Reply:
I followed existing conventions. Feel free to open a new item to modify the conventions throughout.

Reply:
Schema added in https://github.com/inverse-inc/packetfence/pull/2785/commits/e1e09a6cde3ecceea23e7da6eeb30590e011136c

Reply:
The convention in that file seems to be everything together so it would be topauthenticationfailuresbymac

Would still make my eyes bleed but it would follow the "convention" of that file.

IMO, it would be better to just make it underscore separated right away